docs/cluster-registration.md

```diff
@@ -68,10 +68,10 @@ Repeat this step to unjoin any additional clusters.
 Joining additional clusters to a namespaced control plane requires
 providing additional arguments to `kubefedctl join`:
 
-- `--kubefed-namespace=<namespace>` to ensure the cluster has been registered
+- `--kubefed-namespace=<namespace>` to ensure the cluster has been joined
   with the KubeFed control plane running in the specified namespace
 
-To join `mycluster` when `KUBEFED_NAMESPACE=test-namespace` was used for deployment:
+You can join `mycluster` to a control plane deployed in namespace `test-namespace` as follows.
```

Comment:
I would expect `as follows` to be ended with a colon (as per the original text) rather than a period.